src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -161,6 +161,13 @@ private void handleAction(String event,
             } else if (StringUtils.equalsIgnoreCase("pull_request", event)) {
 
                 pr = getPullRequest(payload, gh);
+                GHIssueState state = pr.getPullRequest().getState();
+
+                if (state == GHIssueState.CLOSED) {
+                    LOGGER.log(Level.INFO, "Skip ''{0}'' event on closed PR", event);
```

Comment:
why double `''`

Reply:
> why double `''`

Escapes the quotes.

Already done elsewhere, e.g. https://github.com/hazelcast/jenkinsci-plugin-ghprb/blob/373a6b8fef4853bda73c403ec18fa12d8fa53412/src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java#L157-L160

Reply:
understood. thanks